docs-md/concepts/schemas.md

```diff
@@ -1,17 +1,190 @@
 ---
 layout: page
 title: Schemas in ksqlDB
-tagline: Use schemas in your queries 
+tagline: Defining the structure of your data
 description: Learn how schemas work with ksqlDB
 keywords: ksqldb, schema, evolution, avro
 ---
 
+Data sources like streams and tables have an associated schema. This schema defines the columns
+available in the data, just like a the columns in a traditional SQL database table.
+
+## Key vs Value columns
+
+KsqlDB supports both key and value columns. These map to the data held in the keys and values of the
+underlying {{ site.ak }} topic.
+
+A column is defined by a combination of its [name](#valid-identifiers), its [SQL data type](#sql-data-type),
+and possibly a namespace.
+
+Key columns have a `KEY` namespace suffix. Key columns have the following restrictions:
+  * The can only be a single key column, currently.
+  * The key column must be named `ROWKEY` in the KSQL schema.
+
+Value columns have no namespace suffix. There can be one or more value columns amd the value columns
+can have any name.
+
+For example, the following declares a schema with a single `INT` key column and several value
+columns:
+
+```sql
+ROWKEY INT KEY, ID BIGINT, STRING NAME, ADDRESS ADDRESS_TYPE
+```
+
+## Valid Identifiers
+
+Column and field names must be valid identifiers.
+
+Unquoted identifiers will be treated as upper-case, for example `col0` is equivalent to `COL0`, and
+must contain only alpha-numeric and underscore characters.
+
+Identifiers containing invalid character, or where case needs to be preserved, can be quoted using
+back-tick quotes, for example ``col0``.

```

Comment:
I added this, but forgot to test that it renders the backticks correctly... it doesn't!

```suggestion
back-tick quotes, for example `` `col0` ``.

```